code/game/objects/items/weapons/swords_axes_etc.dm

```diff
@@ -214,7 +214,24 @@
 		src.sharpness = initial(src.sharpness)
 	src.add_fingerprint(user)
 	return
-
+	
+/obj/item/weapon/melee/energy/axe/rusty/attack_self(mob/user as mob)
+	src.active = !( src.active )
+	if (src.active)
+		to_chat(user, "<span class='notice'>The rusty axe is now energised.</span>")
```

Comment:
The codebase standard is to use flexible syntax like `\the`, `[src]`, `\his`, et cetera wherever possible, in order to be more OOP.